app/controllers/admin/institutions_controller.rb

```diff
@@ -1,4 +1,16 @@
 module Admin
   class InstitutionsController < Admin::ApplicationController
+    def import_receivers
+      institution = Institution.find(params[:id])
+      csv_file = params["file"].tempfile
+      csv_content = File.read(csv_file)
+      service = ImportReceiversFromCsv.new(csv_content, institution)
```

Comment:
How about we make the service receive the id and filepath? We could even have named arguments, so it's more explicit. `ImportReceiversFromCSV.new(path: params["file"].tempfile, institution_id: params[:id])`